### ISSUE TITLE
Silently fails on a JSON file that has entries per line

### ISSUE THREAD

**Issue by gbraad** [farling42/obsidian-import-json#39]
I have a JSON file that uses a json-object per line, so it does not use the `[` and `]` to start with.

Unfortunately, no error is returned from the UI and just returns. Only the `non-whitespace` issue is shown on the console.

**Comment by farling42** [comment]
Hi, this sounds like you are trying to something like a foundry db file.

The importer will only read the first object from a JSON file, so it will only manage the first in your file.

**Comment by gbraad** [comment]
Thanks for the info. I just added a `[`, replaced line-endings with `,` and ended with `]`. Worked...

It wasn't a Foundry db file, but similar. It was a large extract of a dictionary. I have decided it is better to write a plugin to do actual lookups as a test the filesystem started to inflate the actual on-disk size for the markdown files and performance was abysmal ;-).

Also handling the uncaught exception (showing the user that an error occurred might prevent raised eyebrows; 'darn, that was fast... oh, nothing happened... why?').
